### PR TITLE
Update ImportError tests with --pre-import

### PR DESCRIPTION
**Pull request by pentschev**
As of https://github.com/dask/distributed/pull/6363, there is a change
in behavior on how plugin errors are raised.

### CODE REVIEW

**Comment by pentschev** [comment]
rerun tests

**Review by galipremsagar — APPROVED** [review]

**Comment by jakirkham** [comment]
This looks good Peter! Should we merge or wait for the Dask release to be cut first?

**Comment by pentschev** [comment]
> This looks good Peter! Should we merge or wait for the Dask release to be cut first?

That's a good question, maybe before merging we should discuss what Dask version(s) will 22.06 support next week.

**Comment by galipremsagar** [comment]
> > This looks good Peter! Should we merge or wait for the Dask release to be cut first?
> 
> That's a good question, maybe before merging we should discuss what Dask version(s) will 22.06 support next week.

Is there an upstream `dask`/`distributed` breaking change coming up in this week's release? or do we want to narrow the rolling window we are supporting now?

**Comment by pentschev** [comment]
> Is there an upstream `dask`/`distributed` breaking change coming up in this week's release? or do we want to narrow the rolling window we are supporting now?

We don't _want_ to narrow it I suppose, but Dask has breaking changes too frequently and that has been the case recently. It would be best to wait and be sure IMO or we can pin to the current `2022.05.0` exclusively.

**Comment by galipremsagar** [comment]
In that case, I'm okay to wait for dask to cut a release and then evaluate(pinning to `>=2022.05.1`)?

**Comment by pentschev** [comment]
Should we tentatively pin to `>=2022.05.0,<=2022.05.1` or should we stick to `=2022.05.1`?

cc @quasiben @jakirkham @galipremsagar @charlesbluca in case any of you have opinions.

**Comment by galipremsagar** [comment]
> Should we tentatively pin to `>=2022.05.0,<=2022.05.1` or should we stick to `=2022.05.1`?
> 
> cc @quasiben @jakirkham @galipremsagar @charlesbluca in case any of you have opinions.

I'm okay with `>=2022.05.0,<=2022.05.1` if dask-cuda has no issues with `2022.05.0`, else would go with `=2022.05.1`

**Comment by pentschev** [comment]
I just had a chat with @quasiben and we're concerned that we have been testing only with Dask/Distributed `main` branch and since there have been so many breaking changes lately it's hard to be certain that `2022.05.0` will indeed be stable. Therefore, I think the best course of action for us is to pin to `=2022.05.1` rather soon to make sure we're in good state as early as possible.

@galipremsagar would you be able to do that nice job you've been always doing and pin `=2022.05.1` in all relevant RAPIDS packages?

**Comment by pentschev** [comment]
rerun tests

Just to be safe things are still fine after https://github.com/dask/distributed/pull/6423

**Comment by galipremsagar** [comment]
> I just had a chat with @quasiben and we're concerned that we have been testing only with Dask/Distributed `main` branch and since there have been so many breaking changes lately it's hard to be certain that `2022.05.0` will indeed be stable. Therefore, I think the best course of action for us is to pin to `=2022.05.1` rather soon to make sure we're in good state as early as possible.
> 
> @galipremsagar would you be able to do that nice job you've been always doing and pin `=2022.05.1` in all relevant RAPIDS packages?

Cool. I'll start opening the pinnings to `=2022.05.1`

**Comment by pentschev** [comment]
Timed out test potentially related to https://github.com/rapidsai/dask-cuda/pull/901#issuecomment-1109547695 , first time I see this timing out since, if it continues to occur we may consider rethinking the test.

**Comment by pentschev** [comment]
rerun test

**Comment by pentschev** [comment]
rerun tests

**Comment by quasiben** [comment]
Everything is passing now.  Are we good to merge ?

**Comment by quasiben** [comment]
@gpucibot merge

**Pull request merged**
